src/Apps/Artist/Routes/WorksForSale/Components/ArtistWorksForSaleEmpty.tsx

```diff
@@ -26,6 +26,20 @@ const ArtistWorksForSaleEmpty: FC<ArtistWorksForSaleEmptyProps> = ({
   const { filters } = useArtworkFilterContext()
 
   const { alertComponent, showAlert } = useAlert({
+    authDialogOptions: {
```

Comment:
**suggestion**: I'm wondering if we need to pass these option into `useAlert` at all. The hook might be able to figure out the `afterAuthAction` attributes on its own. I'm also not sure if we're getting much value out of explicitly setting the contextModule at this level — especially given that we're already tracking an event for clicking on "Create alert".

Reply:
Added a PR against this feature branch with some adjustments for consideration: https://github.com/artsy/force/pull/13133

Reply:
That would be a great improvement and will make the usage of the hook much cleaner 🙏  

I will have a look later and try to integrate it into this PR.

We might need to adjust the logic for when the `alertComponent` is already wrapped with `ProgressiveOnboardingAlertCreateSimple` ([example](https://github.com/artsy/force/blob/e4c6686d623c61e0360c90115fee0b55b0cbf077/src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx#L343)).